File: w08/d02/assessment/express/server_salad.js
var express = require ('express');
var app = express();

app.get('/salad', function(req, res){
	var data = {name: "Basic Salad", ingredients: ["lettuce", "tomatoes", "chicken", "cheese", "croutons"]};
	res.json(data);
})


app.listen(3000)

File: w08/d02/assessment/Sinatra_API/server_salad.rb
require 'sinatra'
require 'json'

get '/salad' do 
	content_type :json
	data = {
		name: "Basic Salad",
	 	ingredients: ["lettuce", "tomatoes", "chicken", "cheese", "croutons"]
	}
	data.to_json	
end


# When you run this server one ruby server_movie.rb and go to the two
# the two extentions and it will send the results to that server